cmd/xcodeUITests.go

```diff
@@ -0,0 +1,134 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/fileutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/goinp/goinp"
+	"github.com/bitrise-tools/codesigndoc/codesigndocuitests"
+	"github.com/bitrise-tools/codesigndoc/xcodeuitest"
+	"github.com/bitrise-tools/go-xcode/utility"
+	"github.com/spf13/cobra"
+)
+
+var xcodeUITestsCmd = &cobra.Command{
+	Use:   "xcodeuitests",
+	Short: "Xcode project scanner for UI tests",
+	Long:  `Scan an Xcode project for UI test targets`,
+
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	RunE:          scanXcodeUITestsProject,
+}
+
+func init() {
+	scanCmd.AddCommand(xcodeUITestsCmd)
+
+	xcodeUITestsCmd.Flags().StringVar(&paramXcodeProjectFilePath, "file", "", "Xcode Project/Workspace file path")
```

Comment:
please attach a screenshot of the help page of the CLI that is realted to this section